supporting-blog-content/evaluating-search-relevance-part-1/retrieve-and-rerank.ipynb

```diff
@@ -774,12 +774,54 @@
    ]
   },
   {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "Finally, in this section we show an implementation of confidence intervals for the `nDCG@10` metric using [bootstrapping](https://en.wikipedia.org/wiki/Bootstrapping_(statistics)) combined with the `percentile` method. It should be noted that this statistic is affected a lot by the number of queries in the dataset i.e. smaller confidence intervals are expected for larger query sets and vice versa",
+   "id": "e9fbbb8760ddc49d"
+  },
+  {
+   "metadata": {},
    "cell_type": "code",
+   "outputs": [],
    "execution_count": null,
-   "id": "0b804730-6969-4854-a56c-72e6b399f0a9",
+   "source": [
+    "def get_ci_with_bootstrapping(scores: list, nr_bootstraps=1000, percentile=95):\n",
+    "    \"\"\"\n",
+    "    Compute confidence intervals using bootstrapping and the percentile method\n",
+    "    Args:\n",
+    "        `scores`: The list of scores to be averaged\n",
+    "        `nr_bootstraps`: The number of bootstrap samples to collect\n",
+    "        `percentile`: The type of confidence interval to compute. It should be a number in (0, 100),\n",
+    "            by default it computes 95% CI\n",
+    "    Returns:\n",
+    "        The confidence interval\n",
+    "    \"\"\"\n",
+    "    estimates = []\n",
+    "    for _ in range(nr_bootstraps):\n",
+    "        sample = np.random.choice(scores, len(scores), replace=True)\n",
+    "        estimates.append(np.mean(sample))\n",
+    "\n",
+    "    half_percentile = (100.0 - percentile) / 2.0\n",
+    "    return np.percentile(estimates, [half_percentile, 100.0 - half_percentile])"
+   ],
+   "id": "3b5563e681b4236"
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "and we can apply it to our results as follows:",
+   "id": "701ed82491660fd6"
+  },
+  {
    "metadata": {},
+   "cell_type": "code",
    "outputs": [],
-   "source": []
+   "execution_count": null,
+   "source": [
+    "ndcg_scores = post_reranking_eval_scores[\"ndcg_cut_10\"]\n",
```

Comment:
When I've used this library I always got the key named as "ndcg_cut@10" I'm sure you tested this but I wonder what the source of the discrepancy is (version?).

Reply:
Not sure about this. From what I see in `pytrec_eval`'s source code the supported formats are either `measure_k` or `measure.k`. Maybe it's a version issue as you say